src/ol_infrastructure/substructure/vault/secrets/__main__.py

```diff
@@ -0,0 +1,28 @@
+import json
+from pathlib import Path
+
+import pulumi_vault as vault
+from bridge.secrets.sops import read_yaml_secrets
+from ol_infrastructure.lib.pulumi_helper import parse_stack
+from pulumi import ResourceOptions
+
+stack_info = parse_stack()
+global_vault_secrets = read_yaml_secrets(
+    Path(f"vault/secrets.{stack_info.env_suffix}.yaml")
+)
+
+# Create the secret mount used for storing global secrets
+global_vault_mount = vault.Mount(
+    "global-secrets-mount",
+    path="secret-global",
+    type="kv-v2",
+    options={"version": 2},
+    description="Storage of global secrets used across our applications.",
+    opts=ResourceOptions(delete_before_replace=True),
+)
+
+vault.generic.Secret(
+    "global-vault-secrets",
+    path=global_vault_mount.path.apply("{}/data".format),
+    data_json=json.dumps(global_vault_secrets),
+)
```

Comment:
This will just write all of the contents of the YAML file to a single path in the mount. If you want it to be one path per top-level key it would need to look like:
```suggestion
for key, data in global_vault_secrets.items():
    vault.generic.Secret(
        "global-vault-secrets",
        path=global_vault_mount.path.apply(lambda path: "{path}/{key}".format(path=path, key=key)),
        data_json=json.dumps(data),
    )
```

Reply:
Yeah I thought about, but don't see the point of splitting it into multiple paths at this point. Majority are global with the exception of Heroku (hoping that we'll be moving away from that in a year or so).

Reply:
I think that the Heroku API keys is a good reason to split it out. We don't want to make those randomly available to apps, even if we are the ones ultimately controlling the reads from Vault.